src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/PostTaskConfigurationTest.java

```diff
@@ -89,6 +89,7 @@ public void should_return_task_configuration_then_expect_task_is_auto_assigned()
             .body("configuration_variables.caseId", equalTo(caseId))
             .body("configuration_variables.securityClassification", equalTo("PUBLIC"))
             .body("configuration_variables.autoAssigned", equalTo(true))
+            .body("configuration_variables.hasWarnings", equalTo(false))
```

Comment:
Can we also update PostConfigureTaskTest Functional test to show that when a task is configured and returned it has the hasWarnings attribute?